p4-16/psa/PSA.mdk

```diff
@@ -58,38 +58,52 @@ interface ports.
 # Target Architecture Model
 
 
+As an analogy, the PSA is to the P4~16~ language as the C standard
+library is to the C programming language.  PSA defines a library of
+types, P4~16~ externs for frequently used constructs such as counters,
```

Comment:
types and externs? We do define a set of data types too.

Reply:
It does say "PSA defines a library of types, ...".  Did you mean you think that "types" should not be mentioned?  If you meant that they should, and it isn't prominent enough, let me know.

Reply:
I guess the comma between types and externs is throwing me off. It is a long sentence that takes a while to absorb. I would probably rephrase it as:
PSA defines a library of types, P4~16~ externs, and a set of "packet paths" that enable you to
write P4 programs that control the flow of packets in a packet switch that has multiple ports. Examples of externs include frequently used constructs such as counters, meters, and registers.

Reply:
Sorry, I merged this, perhaps crossing in flight with your comments.  Please do feel free to edit these additions for clarity.